app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -207,6 +207,11 @@ mautic.config.tab.leadconfig="Contact Settings"
 mautic.config.tab.importconfig="Import Settings"
 mautic.config.tab.contact.list.settings="Contact List Settings"
 mautic.config.tab.columns="Columns"
+mautic.config.tab.contact.merge.settings="Contact Merge Settings"
+mautic.core.config.unique_identifiers_operator="Merge contacts by unique fields with operator"
+mautic.core.config.unique_identifiers_operator.tooltip="If you use more than one unique fields identifiers then you can define the way of the find to merge it. "
```

Comment:
```suggestion
mautic.core.config.unique_identifiers_operator.tooltip="If you use more than one unique field identifiers, you can define the merge behaviour. "
```

Reply:
@RCheesley minor nit, but the default is `en_US`, so shouldn't we be using `behavior` instead of `behaviour`? 😊

Reply:
Ack - yes you are right!